code/modules/clothing/shoes/colour.dm

```diff
@@ -98,27 +98,28 @@
 /obj/item/clothing/shoes/sneakers/orange/attackby(obj/H, loc, params)
 	..()
 	// Note: not using istype here because we want to ignore all subtypes
-	if (H.type == /obj/item/restraints/handcuffs && !chained)
+	if (!chained && H.type == /obj/item/restraints/handcuffs)
 		qdel(H)
 		src.chained = 1
 		src.slowdown = 15
 		src.icon_state = "orange1"
 	return
-
+	
 /obj/item/clothing/shoes/sneakers/orange/allow_attack_hand_drop(mob/user)
 	if(ishuman(user))
 		var/mob/living/carbon/human/C = user
 		if(C.shoes == src && chained == 1)
-			to_chat(user, "<span class='warning'>You need help taking these off!</span>")
-			return FALSE
+			to_chat(user, "<span class='warning'>You start taking off your [src]!</span>")
+			if(!do_after(C,15 SECONDS, src))
+				return FALSE
 	return ..()
 
 /obj/item/clothing/shoes/sneakers/orange/MouseDrop(atom/over)
 	var/mob/m = usr
 	if(ishuman(m))
 		var/mob/living/carbon/human/c = m
 		if(c.shoes == src && chained == 1)
-			to_chat(c, "<span class='warning'>You need help taking these off!</span>")
-			return
+			to_chat(user, "<span class='warning'>You start taking off your [src]!</span>")			
+			if(!do_after(C,15 SECONDS, src))
```

Comment:
```suggestion
			if(!do_after(c,15 SECONDS, src))
```
this one is a lowercase but the top one is uppercase. wow. Whoever coded this before was indeed a true coder.